applications/RomApplication/python_scripts/hrom_training_utility.py

```diff
@@ -132,6 +134,8 @@ def CreateHRomModelParts(self):
     @classmethod
     def __GetHRomTrainingDefaultSettings(cls):
         default_settings = KratosMultiphysics.Parameters("""{
+            "original_number_of_elements": 10,
```

Comment:
```suggestion
            "original_number_of_elements": 0,
```
I'd default this to zero to then check that a valid value is provided, in other words, if after the defaults check the `original_number_of_elements` is still 0 we throw an error.